cg/clients/freshdesk/freshdesk_client.py

```diff
@@ -71,3 +81,23 @@ def _configure_retries(session: Session) -> None:
         )
         adapter = HTTPAdapter(max_retries=retry_strategy)
         session.mount("https://", adapter)
+
+    @staticmethod
+    def create_new_ticket_attachment(content: dict, file_name: str) -> dict:
+        return {
+            file_name: TEXT_FILE_ATTACH_PARAMS.format(
+                content=WriteStream.write_stream_from_content(
+                    content=content, file_format=FileFormat.JSON
+                )
+            )
+        }
+
+    @staticmethod
+    def create_connecting_ticket_attachment(content: dict) -> TemporaryDirectory:
+        directory = TemporaryDirectory()
+        WriteFile.write_file_from_content(
+            content=content,
+            file_format=FileFormat.JSON,
+            file_path=Path(directory.name, "order.json"),
+        )
```

Comment:
This logic does not belong here, the only responsibility of the `FreskAPIClient` is to communicate with the fresh desk web API. It should not be dealing with files etc.

I suggest you move this logic out into the `TicketHandler` instead and remove the `attachments: list[Path]` parameter from the `create_ticket` method and just ensure that the attachments are in the `CreateTicket` model.